src/pydata_sphinx_theme/assets/styles/content/_api.scss

```diff
@@ -104,3 +104,27 @@ span.highlighted {
   position: relative;
   background-color: var(--pst-color-target);
 }
+
+/*******************************************************************************
+* Styling for autosummary titles like "parameters" and "returns"
+*/
+
+// the API selector
+// from https://github.com/pradyunsg/furo/blob/main/src/furo/assets/styles/content/_api.sass#L6)
+dl[class]:not(.option-list):not(.field-list):not(.footnote):not(.glossary):not(.simple) {
+  dd {
+    margin-left: 2rem;
+  }
+
+  dl.field-list {
```

Comment:
Why move this our of `_lists.scss`? Wouldn't this also apply to non-API field lists too?

Reply:
I guess the question is whether this would also apply to field lists like in the kitchen sink here:

https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/lists.html#field-list

Reply:
I'm happy with whatever. If it also applies to `field-lists`, but primarily applies to API docs, then let's just add a comment like `// Note: this also applies to other field lists but is primarily used for API docs`

Reply:
wait, qui proco here. This modification only applies to API related `dt` and `dl`, look at the first very long encapsulating selector, it includes `:not(.glossary)`

Reply:
> This modification only applies to API related `dt` and `dl`, look at the first very long encapsulating selector, it includes `:not(.glossary)`

Is there a reason *not* to apply this change to other field lists besides APIs? Consistency seems good to me.

Reply:
APIs generated by numpydocs/napoleon were the only problematic ones. So I would not take the risk to change all the others if they are working nicely